.github/workflows/test.yml

```diff
@@ -102,6 +102,28 @@ jobs:
           make test-integration
         if: env.GIT_DIFF
 
+  integration_tests:
```

Comment:
we should use the `get-diff-action` to only run the integration tests when the go files have been updated (see above)